telemetry/Tracks/class-tracks-client.php

```diff
@@ -0,0 +1,187 @@
+<?php
```

Comment:
Unless we're using uppercase elsewhere, we should rename the folder to `tracks` (lowercase `t`).